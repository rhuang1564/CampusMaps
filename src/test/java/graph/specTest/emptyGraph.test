# Tests that a newly created graph is empty.

# Create a graph
CreateGraph A

# Print the nodes in the graph
ListNodes A