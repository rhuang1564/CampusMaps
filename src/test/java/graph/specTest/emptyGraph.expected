# Tests that a newly created graph is empty.

# Create a graph
created graph A

# Print the nodes in the graph
A contains: